plugin/lsp.vim

```diff
@@ -30,6 +30,7 @@ let g:lsp_preview_autoclose = get(g:, 'lsp_preview_autoclose', 1)
 let g:lsp_preview_doubletap = get(g:, 'lsp_preview_doubletap', [function('lsp#ui#vim#output#focuspreview')])
 let g:lsp_peek_alignment = get(g:, 'lsp_peek_alignment', 'center')
 let g:lsp_preview_max_width = get(g:, 'lsp_preview_max_width', -1)
+let g:lsp_hover_conceal = get(g:, 'lsp_hover_conceal', 1)
```

Comment:
Do you think it would be better to disable this by default?

Reply:
We can enable it by default or look at the user fold settings i.e. `nofoldenable`?